docs/exceptions/exceptions-logs.md

```diff
@@ -52,4 +58,13 @@ The table below indicates which attributes should be added to the
 Same as [Trace Semantic Conventions for Exceptions - Stacktrace
 Representation](exceptions-spans.md#stacktrace-representation).
 
+## Guidelines for Recording Exceptions in Spans, Logs, and Events
+
+When it comes to recording exceptions, it's crucial to distinguish between Spans, Logs, and Events:
+
+* Spans and Events: Exception recording in Spans and Events follows a similar pattern. Both utilize the event name `exception`. Exceptions MAY be recorded within Span Events if they occur during the span's lifecycle.
```

Comment:
```suggestion
* Spans and Events: Exception recording in Spans and Events follows a similar pattern. Both utilize the event name `exception`. Exceptions SHOULD be recorded within Span Events if they occur during the span's lifecycle.
```

Is there any reason why we'd diverge from the stable spec statement that makes this a SHOULD here in semconv?
https://opentelemetry.io/docs/specs/otel/trace/exceptions/